homeassistant/components/met/coordinator.py

```diff
@@ -117,7 +116,11 @@ async def _async_update_data(self) -> MetWeatherData:
         try:
             return await self.weather.fetch_data()
         except Exception as err:
```

Comment:
Okay so 2 things
1. Ideally we don't catch raw Exceptions outside of the config flow and outside of tasks, so IMO we should just remove this except. This might uncover some bugs like a ValueError or AttributeError, but we should watch for those and properly fix the code
2. You're now raising a translated exception to be caught and to be raised again as translated exception, I think we can just do this once